### ISSUE TITLE
Error on unnamed imports

### ISSUE THREAD

**Issue by turadg** [nadeesha/ts-prune#28]
If you import for a side-effect without assigning, it causes an error at `.getText()` because `decl.getImportClause()` returns undefined.

E.g.,
```
import 'mock-local-storage';
```

**Comment by OliverJAsh** [comment]
Also ran into this issue. I tried to fix it by conditionally calling the method:

``` js
      const newLocal = decl
        .getImportClause();
        var containsWildcardImport = newLocal !== undefined ? newLocal
            .getText()
            .includes("*") : false;
```

However then I ran into another issue:

```
/Users/oliverjash/Development/unsplash-web/node_modules/ts-prune/lib/analyzer.js:40
                file: decl.getModuleSpecifierSourceFile().getFilePath(),
                                                         ^

TypeError: Cannot read property 'getFilePath' of undefined
    at /Users/oliverjash/Development/unsplash-web/node_modules/ts-prune/lib/analyzer.js:40:58
    at Array.forEach (<anonymous>)
    at emitDefinitelyUsed (/Users/oliverjash/Development/unsplash-web/node_modules/ts-prune/lib/analyzer.js:32:34)
    at /Users/oliverjash/Development/unsplash-web/node_modules/ts-prune/lib/analyzer.js:70:9
    at Array.forEach (<anonymous>)
    at Object.exports.analyze (/Users/oliverjash/Development/unsplash-web/node_modules/ts-prune/lib/analyzer.js:68:30)
    at Object.exports.run (/Users/oliverjash/Development/unsplash-web/node_modules/ts-prune/lib/runner.js:18:16)
    at Object.<anonymous> (/Users/oliverjash/Development/unsplash-web/node_modules/ts-prune/lib/index.js:5:10)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
```

**Comment by nadeesha** [comment]
Thanks for reporting this. #31 fixes this on `v0.2.1`.
